frontend/src/app/selectors/news.js

```diff
@@ -58,26 +65,31 @@ const makeNewsAgeSelector = includeStale => (newsUpdates, twoWeeksAgo) => {
     return includeStale ? dt < twoWeeksAgo : dt >= twoWeeksAgo;
   });
 
-  if (includeStale) return result;
-  // if includeStale return before filtering out updates which have
-  // been seen.
+  if (!lastViewedDate) {
+    return result;
+  }
+
+  // If lastViewedDate is supplied, further filter result by that cut-off date
   return result.filter(update => {
     const dt = new Date(update.published || update.created).getTime();
```

Comment:
This isn't your change, but I'm realizing that here and above on line 64 we shouldn't be checking for `update.created` it should only be `published`

Reply:
I forgot to update that in the last pr